### PR TITLE
fix #75810 123films.cc

### LINKED ISSUE #75810: 123films.cc
### Issue URL (Ads)
 [https://123films.cc/the-sparks-brothers/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2F123films.cc%2Fthe-sparks-brothers%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.6 nightly 19
Browser: | Opera
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (180), <br>Block Push API, <br>Block Location API
DNS filtering: | server: `https://dns-unfiltered.adguard.com/dns-query`<br>filters: `User rules`
Filters: | <b>Ad Blocking:</b><br/> EasyList, <br/>Peter Lowe's Blocklist, <br/>AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/> EasyPrivacy, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> Adblock Warning Removal List, <br/>I don't care about cookies, <br/>EasyList Cookie List, <br/>AdGuard Annoyances<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist, <br/>Spam404, <br/>NoCoin Filter List<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish, <br/>AdGuard Russian
Custom filter subscriptions: | https://adguardteam.github.io/AdGuardSDNSFilter/Filters/filter.txt

### PR DESCRIPTION
**Pull request by wout-codes**
#75810

Fixes fake download buttons and several pop-ups in different streams.
You can find more popuups at `https://123films.cc/my-brothers-wife/`. Please use AdGuard for Windows.

### CODE REVIEW

File: EnglishFilter/sections/general_extensions.txt [hunk 1]
```diff
@@ -167,7 +167,7 @@ lookmovie.ag#%#//scriptlet("abort-on-property-write", "zfgproxyhttp")
 ! #%#//scriptlet("prevent-setInterval", "[$."), #%#//scriptlet("prevent-setTimeout", "[$."), #%#//scriptlet("prevent-addEventListener", "", "[$.")
 movies123.pk,manga-raw.club#%#//scriptlet("prevent-setInterval", "[$.")
 movies123.pk,123movieslinks.live,kickass.ws,kickass.love,kissanime.nz,animekisa.tv,repackov.com#%#//scriptlet("prevent-setTimeout", "[$.")
-movies123.pk,sportstreamtv.live,uwatchfree.tech,beinmatch.tv,fmovies.to,samehadaku.vip,kickass.ws,kickass.love,animafan.biz,manga-raw.club,kissanime.nz,animekisa.tv,repackov.com#%#//scriptlet("prevent-addEventListener", "", "[$.")
+123films.cc,movies123.pk,sportstreamtv.live,uwatchfree.tech,beinmatch.tv,fmovies.to,samehadaku.vip,kickass.ws,kickass.love,animafan.biz,manga-raw.club,kissanime.nz,animekisa.tv,repackov.com#%#//scriptlet("prevent-addEventListener", "", "[$.")
```
Comment (Alex-302) [inline EnglishFilter/sections/general_extensions.txt:170]: we already have the rule for script: `#%#//scriptlet("abort-current-inline-script", "String.fromCharCode", "delete window")`

Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:170]: Doesn't work. I reproduced pop-ups.

Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:170]: @Alex-302 What should I do?

Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:170]: Ok I will add it there.

**Review by zloyden — APPROVED** [review]

**Comment by zloyden** [comment]
@wout-codes don't add rules at the beginning, fixed conflicts.

**Pull request merged**